espresso/BasicSample/app/build.gradle

```diff
@@ -24,7 +24,7 @@ android {
 
 dependencies {
     // App dependencies
-    compile 'com.android.support:support-annotations:22.1.0'
```

Comment:
Revert and add:
allprojects {
    repositories {
        jcenter()
    }

    configurations.all {
        resolutionStrategy.force 'com.android.support:support-annotations:22.1.0'
    }
}

to espresso/BasicSample/build.gradle

Would be great if you could apply the same fix to all the individual samples which depend on support-annotations